documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1140,48 +1140,359 @@ sqlplus sys/top_secret@//localhost:1521/ORCLCDB as sysdba
+* Oracle Database is installed, and is {link-prefix}:{link-oracle-connector}#setting-up-oracle[configured to work with the {prodname} connector].
+* You have a copies of the Oracle JDBC driver and the XStream API JAR.
+Due to licensing requirements, the {prodname} Oracle Connector does not ship with these files.
++
+For more information, see {link-prefix}:{link-oracle-connector}#obtaining-oracle-jdbc-driver-and-xstreams-api-files[Obtaining the Oracle JDBC driver and XStream API files].
+
+.Procedure
+
+. Download the {prodname} https://repo1.maven.org/maven2/io/debezium/debezium-connector-oracle/{debezium-version}/debezium-connector-oracle-{debezium-version}-plugin.tar.gz[Oracle connector plug-in archive].
++
+[NOTE]
+====
+As stated in the preceding prerequisites, the {prodname} Oracle connector does not include the Oracle JDBC driver or the XStream API JAR.
+====
+. Extract the files into your Kafka Connect environment.
+
+. Add the directory with the JAR files to {link-kafka-docs}/#connectconfigs[Kafka Connect's `plugin.path`].
+
+  . {link-prefix}:{link-oracle-connector}#oracle-example-configuration[Configure the connector] and {link-prefix}:{link-oracle-connector}#oracle-adding-connector-configuration[add the configuration to your Kafka Connect cluster.]
+. Restart your Kafka Connect process to pick up the new JAR files.
+endif::community[]
 
-Due to licensing requirements, the {prodname} Oracle Connector does not ship with the Oracle JDBC driver and the XStream API JAR.
-You can obtain them for free by downloading the https://www.oracle.com/database/technologies/instant-client/downloads.html[Oracle Instant Client].
+ifdef::product[]
+To deploy a {prodname} Oracle connector, you add the connector files to Kafka Connect, create a custom container to run the connector, and then add connector configuration to your container.
+For details about deploying the {prodname} Oracle connector, see the following topics:
+
+* xref:obtaining-the-oracle-jdbc-driver[]
+* xref:deploying-debezium-oracle-server-connectors[]
+* xref:descriptions-of-debezium-oracle-server-connector-configuration-properties[]
+endif::product[]
 
-Extract the archive and navigate to the Instant Client directory.
-Copy `ojdbc8.jar` and `xstreams.jar` (not needed when using the LogMiner-based capture implementation) into Kafka Connect's _libs_ directory.
-Lastly, create an environment variable, `LD_LIBRARY_PATH`, that points to the instant client directory, as shown below:
+// Type: procedure
+// Title: Obtaining the Oracle JDBC driver
+// ModuleID: obtaining-the-oracle-jdbc-driver
+[id="obtaining-oracle-jdbc-driver-and-xstreams-api-files"]
+=== Obtaining the Oracle JDBC driver and XStream API files
 
+ifdef::community[]
+Depending on your Oracle environment, you must have either the Oracle JDBC driver (`ojdbc8.jar`) or XStream API (`xstreams.jar`) to run the {prodname} Oracle connector.
```

Comment:
This sentence is a tad misleading.  The `ojdbc8.jar` is required regardless of which adapter the connector config uses.  The `xstreams.jar` is needed only if the connector config uses the `xstream` adapter.

Reply:
Ah, ok. Thanks, I (obviously) wasn't clear on that.

Reply:
OK, fixed.